safe/definitions/hazard_classifications.py

```diff
@@ -220,9 +220,9 @@
     ]
 }
 
-flood_hazard_classes = {
-    'key': 'flood_hazard_classes',
-    'name': tr('Flood classes'),
+flood_wet_dry_hazard_classes = {
+    'key': 'flood_wet_dry_hazard_classes',
```

Comment:
AFAIK, changing an existing key is going to break every existing datasets using the old `flood_hazard_classes` hazard classification.
I expect some python exception to be raised as classes are not the same. Maybe we should check first.